client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts

```diff
@@ -76,6 +76,7 @@ export interface BoostData {
 
 export interface Site {
 	blog_id: number;
+	blogname: string;
```

Comment:
Is there a reason we are using `blogname` and not `blog_name` (since the above line is `blog_id`) ?

Reply:
It's the field name given by the endpoint:

<img width="673" alt="image" src="https://github.com/Automattic/wp-calypso/assets/9832440/237ab9ee-e0af-4aa4-8220-d5c7d42a688c">